### ISSUE TITLE
Increase coverage

### ISSUE THREAD

**Issue by drkrillo** [drkrillo/good-first-issues#36]
Tests need to be implemented using pytest to rise the coverage %,

**Comment by drkrillo** [comment]
Thank You @ChristinaRaichel !

**Comment by taneeshamadhu18** [comment]
hey i would love to contribute too

**Comment by drkrillo** [comment]
> Hi @drkrillo I am new to open source. Did you mean pytest/puretest for increasing test coverage?

Would You like to contribute ?

**Comment by drkrillo** [comment]
> hey i would love to contribute too

@taneeshamadhu18  if the other user doesnt take it, i'll let you know. Is that OK ?

**Comment by ChristinaRaichel** [comment]
Hi @drkrillo, I apologize, I'm tied up with other tasks at the moment. Hopefully, I can contribute to the code later

**Comment by drkrillo** [comment]
@taneeshamadhu18 Are you still interested in taking the issue?

**Comment by taneeshamadhu18** [comment]
Yess yess I amm

**Comment by taneeshamadhu18** [comment]
Yes I am!

**Comment by drkrillo** [comment]
@taneeshamadhu18 great! go ahead and take it.

Just fork the repo, work on whatever tests you want to add, and make an MR to dev branch.

**Comment by taneeshamadhu18** [comment]
Thank you

**Comment by Eswar133** [comment]
hey @drkrillo I am new to open-source contributions. Please let me know if there are any beginner friendly tasks are there

**Comment by lohith1511** [comment]
i am new to open source, still the issue is in open .So, please let me know that have any chance to contribute this issue.

**Comment by drkrillo** [comment]
Yes @lohith1511 feel free to take it. Do you know how to start? Tests should be added to increase the coverage

**Comment by drkrillo** [comment]
Yes @P3st83 ! Go ahead
